QuantLib.vcxproj

```diff
@@ -1107,6 +1107,7 @@
     <ClInclude Include="ql\math\randomnumbers\boxmullergaussianrng.hpp" />
     <ClInclude Include="ql\math\randomnumbers\centrallimitgaussianrng.hpp" />
     <ClInclude Include="ql\math\randomnumbers\faurersg.hpp" />
+    <ClInclude Include="ql\math\randomnumbers\fractionalnoisersg.hpp" />

```

Comment:
It might be a good idea to put everything in the `experimental` directory to start with in order to give yourself the option of changing the public interface in the near future if necessary.